packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -627,13 +627,16 @@ describe.skip('Line chart - Subcomponent Event', () => {
   testWithWait(
     'Should render events with defined data',
     LineChart,
-    { data: simplePoints, eventAnnotationProps, tickValues, tickFormat: '%m/%d' },
+    { data: simplePoints, eventAnnotationProps: eventAnnotationProps },
     container => {
       // Arrange
-      const event = screen.queryByText('3 events');
-      // Assert
-      expect(event).toBeDefined();
-      fireEvent.click(event!);
+      const svg = container.querySelector('svg');
```

Comment:
dont use queryselector with RTL. refer RTL documentation on how to get elements to validate in RTL

Reply:
We have updated the querySelector with getElementsByTagName.